test/parallel/test-string-decoder-end.js

```diff
@@ -8,13 +8,13 @@ const assert = require('assert');
 const SD = require('string_decoder').StringDecoder;
 const encodings = ['base64', 'hex', 'utf8', 'utf16le', 'ucs2'];
 
-const bufs = [ '☃💩', 'asdf' ].map(function(b) {
+const bufs = [ '☃💩', 'asdf' ].map((b) => {
```

Comment:
Optional: This could just be `(b) => Buffer.from(b)`. (The current code base does have a mixture of using `{...}` and `return` when it's not required vs. omitting them, so either should be fine, but omitting them is definitely more common.)